x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -363,12 +363,15 @@ export const ActionsConnectorsList: React.FunctionComponent = () => {
                 ),
               });
               // Refresh the actions from the server, some actions may have beend deleted
-              loadActions();
+              await loadActions();
             }
           }
           setConnectorsToDelete([]);
         }}
-        connectorsToDelete={connectorsToDelete}
+        apiDeleteCall={deleteActions}
+        idsToDelete={connectorsToDelete}
+        singleTitle={'connector'}
```

Comment:
Not sure if these should be i18n'd here, or where used ... seems like it should be here, but maybe it's easier for the translations to do it where they are used.  Same with the alerts.